tox.ini

```diff
@@ -94,3 +94,13 @@ deps=
 changedir=docs/source
 commands=
     sphinx-build -W -b html . {envtmpdir}/html
+
+
+[testenv:py37-nodeps]
+deps=
+    {[base]deps}
+
+commands=
+    ; without lightning as it is optional
+    pip install -e .
+    bash _ci/runtests_nodeps.sh {posargs: eli5 tests}
```

Comment:
(refers to review above, forgot to add this comment).
I wonder if ``commands={[testenv:py35-nodeps]commands}`` will work here, since we run the same commands